tests/test_validators.py

```diff
@@ -138,6 +138,17 @@ class TestEnum(enum.Enum):
     assert encoder.to_json(TestEnum.B) == 'b'
 
 
+def test_outgoing_message_decimal_encoder():
+    encoder = validators.OutgoingMessageDecimalEncoder()
+
+    assert encoder.from_json(1.123456789) == 1.123456789
+
+    with pytest.raises(errors.TypeConstraintViolationError):
+        encoder.to_json('well thats not a float')
+    encoder.to_json(1.12) == 1.12
+    encoder.to_json(1.123456789) == 1.123456
+
```

Comment:
You should also test floats with exponents as I think the current method won't work with values like `12.123456e+32`.